src/SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -18,6 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using SonarQube.Client.Api.V7_20;
```

Comment:
I don't think this is desired in this file. As it's meant to work with various ranges of versions.
```suggestion
```

Reply:
This is due to the fact that I didn't add `V7_20.` prefix to `GetIssuesWithComponentSonarQubeRequest`. Compare it with `V7_20.GetIssuesRequestWrapper`, for example. I can fix it, if you think it would make it clearer

Reply:
I'd say that every class usage in this file should have it's version. So files from a globally imported specific version V7_20 do not leak to unexpected lines.